##Library Construction
###General notes
* Elution buffer: make components for yourself, EDTA from Saisai (fridge #50 lower right corner)
* Polysome buffer: use components of Saisai for preparation of this buffer
* Commercial RNase –free water on Saisai’s bench
* Aliquot in 50 ml tubes before use
* riboseq box in 4°C fridge #50 bottom left corner
* -20°C freezer #094 bottom left and middle 
* Use special batch of A549 cells from LN2 (which one? Saisai)
* Infected Cells should 80-90% confluent at time of collection
* 2x 10ml plate for each timepoint
* Use new CHX for lysate buffer
* Use old CHX powder in 4°C for PBS and sucrose
* Prepare 60% sucrose stock
  * Use this to prepare the 45%, 15% and 1M Sucrose solutions
####Protocol

* Wash 2x with ice cold PBS+CHX
* 1/1000 dilution CHX
* Tilt plates in ice for easy removal of PBS+CHX
* Add 400 μl lysis buffer to first plate
* Stored in -20°C freezer (#53)
* 400 μl lysis buffer for each timepoint
* Polysome buffer, 2% triton, CHX (1/500)
* 100 μg/ml stock solution CHX (store at 4°C)
* Scrape cells
* Transfer lysate to second plate with same collection time
* Transfer lysate to 1.5 ml Eppendorf
* Incubate on ice for 15 min.
* Store at -80°C

**Day2**
•	Transfer 100 μl lysate of each timepoint to separate Eppendorf for RNAseq
o	Keep on ice
•	Centrifuge lysate for Riboseq at 13000 rpm small rotor, 4°C, 10 min.
•	transfer supernatant to new Eppendorf and discard pellet
•	add RNaseI 4 μl/plate (so 8 μl in this case) to lysate
o	stored in -20°C freezer
•	incubate in cold room on rotator for 1 hour (fix Eppendorfs with tape)
•	add 4 volumes of Trizol LS reagent to RNAseq aliquots (so total volume 500 μl)
o	stored in flammable closet under chemical hood
•	Incubate 5 min. at RT then store at -80°C freezer 
•	Place Riboseq samples on ice after incubation 
•	Add 4 μl SUPERase In to each sample
•	Proceed to Sucrose gradient protocol
 
•	Sucrose Cushion
o	Make 1M sucrose from 60% stock
•	60% sucrose = 60 g/ 100 ml
•	1M sucrose = 34.2 g/ 100 mL
•	dilute 5.67 mL 60% stock in 4.33 ml polysome buffer to make 10 ml
o	Add 1 ml 1 M sucrose containing 100 µg/mL CHX (1/1000 dilution) in the ultracentrifuge tube 
•	Thickwall, Polycarbonate, 3.5 mL, 13 x 51 mm (Beckman 349622)
o	add <600 µl cell lysate on the top
•	If the lysate is more than 600 µL, split into two tubes
o	Centrifugation at 54,000rpm for 6.5 h with the SW 55Ti
•	pellet will be transparent
•	Talk to Wendy (Parrish lab for centrifuge introduction)
o	Remove the sucrose 
o	Wash pellet with with polysome buffer without dislodging it
o	Dissolve the pellet in Trizol
o	Store samples at -80 °C for future library construction
 
•	Sucrose Gradient
o	Prepare ~7 ml of both 45% and 15% sucrose solution per sample
•	By diluting premade 60% sucrose
Final Con.	Total Volume	Volume of 60% sucrose	Volume of polysome buffer
15%	15 mL	3.75 mL	11.25 mL
45%	15 mL	11.25 mL	3.75 mL
15%	25 mL	6.25 mL	18.75 mL
45%	25 mL	18.75 mL	6.25 mL

o	Mark the centrifugation tubes with the mark block
•	Tube, Thinwall, Polypropylene, 13.2 mL, 14 x 89 mm (Beckman)
o	Fill the tubes with 15% sucrose solution first with needle and syringe-fill to just slightly above the marked line
o	Add 45% sucrose solution starting at the bottom of the tube slowly to the marked line 
•	lift up slowly as you add to displace weight 
•	Should get a “bulb” at the top of the tube with the 15% bubbling almost over.
o	Close of tube with black rubber stopper, keeping the air hole at the top during closing
o	Turn on the gradient master
•	 make sure the gradient master is level
o	Put tubes into the magnetic holder and place it on the gradient master
o	Program
•	go to gradient->recent (double check that this is what you want)->sw41->use->run. 
•	If a different gradient is required, adjust accordingly
o	While gradient master is running, clean the needle with methanol and DEPC
o	When gradient is done, remove stoppers
o	Remove 600 µl off top of tubes with tips
o	Gently load 550 µl of sample to top
o	Weigh the tubes so that they are balanced (within 0.05g) for centrifugation
o	Slowly add add samples into the holders of the SW41 centrifuge rotor
o	centrifuge at 38000 rpm for 2 hours and 38 min at 4 °C
•	Fractionation
o	During centrifugation set up fractionator
•	Turn on the pump to manual. 
•	Place the small tubing into 50 ml methanol, and reverse rapid into the syringe of the pump. 
•	Remove all air bubbles from syringe and tubing by tilting the machine carefully vertically and forward pump out bubbles
•	connect tubing to the needle. 
•	Slowly pump till methanol comes out of the needle and all bubbles are gone. 
•	Turn bottom of apparatus to puncture the bottom of the empty sample tube. 
•	Try to get the needle in the middle of the tube. 
•	Continue Puncturing tube until both black lines on needle are visible
•	pump the methanol into the tube until methanol reaches the wast
•	Remove methanol from tubing by reverse pumping.
•	Than transfer methanol back to 50ml tube storage tube
•	Repeat this twice with DEPC treated water
o	Fractionating sucrose gradients
•	Prepare ice bucket with premade holes
•	After centrifugation, remove tubes from holders with a forceps and place them on ice
•	Prepare centrifuge by filling syringe with ±25 ml of 60% sucrose solution
•	Attach seal apparatus on sample tube. 
•	Pump forward on normal speed to remove all bubbles from needle
•	place the needle in line with the sample tube until the apparatus is sealed tight
•	slowly wind the needle into sample until both black lines on needle are visible
•	Check if graph machine is warmed up (light green)
•	Remove cap of red chart marker
•	Set needle marker at about 30 on graph
•	Place Eppendorf tube in collection platform
•	Turn Brandel pump to “remote”
•	Settings on “forward”, “normal”, and “1.5”
•	Press run
•	Mark the graph paper as the peaks come out to correspond with tubes. 
•	Label tubes and place on ice
•	When profiling is done or ready to be aborted, go to manual on Brandel go and reverse sample back.
•	Forward pump into a waste tube
o	Clean fractionator system
•	Remove graph
•	Cap graph marker
•	go back and forth with methanol 3x trough tubing
•	Rinse system with DEPC-treated water 
 
Day3
•	Add 500 μl chloroform to both RNA and Riboseq samples
•	Centrifuge 13000 rpm/min, 15min, 4°C
•	Transfer top layer to new Eppendorf
•	Add 500 μl of ice-cold isopropanol 
o	Add 4 μl of glycogen to Riboseq samples for visualizing pellet
•	Incubate at RT for 10 min (Saisai does 30min)
•	Centrifuge 13000 rpm/min, 15min, 4°C
•	Wash with 70% ethanol (-20°C) 
o	Shake Eppendorf till pellet floats
o	Centrifuge 13000 rpm/min, 5min, 4°C
•	Dissolve RNAseq pellet in 10 μl and riboseq in 15 μl MQ H2O
•	Measure concentrations with Nanodrop 2000
o	Concentrations normally around 1500-2000 ng/μl
o	If total amount is less than 18 μg, use protocol for low amount RNA
•	Prep Riboseq samples
o	E.g.: 1531 μg/μl; need ±10 μg
•	So take 7 μl of riboseq sample and add complete to 11 μl with RNAse-free water
•	Store at -80°C
•	Fragmentation of RNAseq samples
o	Adjust the RNA concentration to 1 μg/μl with RNAse-free water. Set up the following fragmentation in a thin-walled 200 μl PCR tube. Vortex and spin down tube
o	Reaction: 18 μl of RNA (1 μg/μl), 2 μl fragmentation buffer 10x, 
•	Incubate tube at 94°C for 5 min in thermal cycler block with heated lid on
•	Remove PCR tube from block and immediately add 2 μl of 0.5M EDTA
•	Vortex, spin down, put on ice
o	In case of low amount of RNA
•	18 μl of RNA (4 μg total amount of RNA), 2 μl of fragmentation buffer
•	94°C for 3 min
o	Repeat previous steps for each batch of five tubes until all RNA has been fragmented
o	Collect content of all tubes in 1.5 ml eppendorfs and for 400 μl  of fragmentation mixture add 40 μl of 3M sodium acetate (pH 5.2), 4 μl glycogen (100 μg/ml final) and 890 μl of 100% ethanol
o	Mix content and incubate at -80°C O/N
 
 
Day 4
•	RNAseq samples: centrifuge at 13000 rpm for 20 min. at 4°C
•	Wash with 70% ethanol: 13000 rpm for 10 min at 4°c 
•	Disolve pellet in 11 μl RNase free water 
•	Dephosphorylation for both RNAseq and Riboseq samples
o	Make mastermix for all the samples (e.g. 8 samples)
o	Total RNA Samples (RNA was dissolved in 11 µL H2O) were dephosphorylated in a 15 µL reaction containing 
•	1 × T4 polynucleotide kinase buffer (1.5 µL)
•	10 U SUPERase_In (0.5 µL)
•	20 U T4 polynucleotide kinase (2 µL) (NEB)
o	Dephosphorylation was carried out for 2 hour at 37°C (program 1 on heatblock)
•	Size selection
 
o	Dephosphorylated samples were mixed with 15 μl of 2 × Novex TBE-Urea sample buffer (Invitrogen)
•	Use 200 μl pipette
o	Samples heated at 70°C for 3 min
o	Load samples on a 15% polyacrylamide TBE-urea gel
•	The samples are separated by 1 × loading dye
•	Prepare this by diluting 2x loading buffer with RNase-free water
•	0.5 μl 10bp ladder was used 
•	Clean well by pipetting before loading samples
•	Loading sequence: 10bp ladder, Sample 1, loading dye, Sample 2, loading dye, Sample 3, loading dye, Sample 4, loading dye, 1 μl 28 nt RNA
o	Run gel at 240V for 1.5h
•	Staining gel with SYBR Gold (Invitrogen, diluted in running buffer 1:10,000) in dark for 15-20 min (saisai does 5 min) on shaker (speed 3) SYBR Gold kept in aluminum rapped 50ml aliquots 
•	Transfer gels and staining solutions into plastic containers that will be used for the same gels during the rest of the protocol
o	Gel bands containing RNA species corresponding to 40-60 bp (RNAseq) or 28 bp (Riboseq) (between two clear bands, 26nt-34nt) were excised
   
RNAseq samples (left) & Riboseq samples (right)
•	Take pictures of gels before and after band excision with Biorad Chemidoc MP, Imagelab 5.1 software (room 223)
o	Don’t forget sign in sheet
o	Safe high resolution pictures using export for publication
o	New protocol→gel imaging
•	Gel pieces are physically disrupted using centrifugation through the holes of the tube (centrifuge at 13000 rpm for 10 min or more.).  
o	Small eppie with holes made with needle 22G safetyguard
o	Put inside 1.5 eppie 
•	Label 1.5 eppie on the side!
•	RNA fragments are dissolved by soaking overnight in 400 μl RNA gel elution buffer (300 mM NaOAc pH 5.5, 1 mM EDTA, 0.1 U/mL SUPERase_In)
o	1ml of 3 M NaOAc/10ml, 20 μl 0.5M NaOAc/10ml, 50 μl SUPERase_In/10ml
o	Incubate in cold room on rotator O/N
 
Day 5
•	gel debris was removed using a Spin-X column (Corning) 
o	12000 rpm 2 min
•	Eluent was transferred to a new 1.5 mL tube 
•	RNA was purified using ethanol precipitation
o	See above
o	(400 µl liquid + 890 µL ice cold ethanol + 40 µl 3M NaAc pH 5.2 + 4 µL glycogen at -20°C for 30 min, 14000 rpm 10 min, 500 µL 70% ethanol wash, 14000 rpm 5 min)
•	Purified RNA fragments were resuspended in 5.2 µL H2O in 1.5 ml Eppendorf tubes
•	Poly-(A) tailing
o	reaction was performed in a 8 µL system with 1 × poly-(A) polymerase buffer (0.8 µl), 1 mM ATP (1 µl), 0.75 U/µL SUPERase_In (0.4 µl), and 3 U E. coli poly-(A) polymerase (0.6 µl) (NEB)
o	mastermix was made for all samples (for 8 samples multiply amounts by 8.5)
o	2.8 µl of the mastermix was added to each sample
•	Use tips with filter
o	Tailing reaction at 37°C for 45 min
•	Program 1 of heatblock
•	TR-PCR: cDNA synthesis
o	For reverse transcription, the following oligos containing barcodes were synthesized:
•	MCA02: 5’-pCAGATCGTCGGACTGTAGAACTCTCAAGCAGAAGACGGCATACGATTTTTTTTTTTTTTTTTTTTVN-3’;
•	LGT03:5’-pGTGA……-3’;
•	HTC: 5’-pAGGA……-3′ 
•	YAG: 5’- pTCGA……-3′;
•	SCT01:5’-pCTGA……-3’ (this oligo was used for 28 nt).
•	Barcode nucleotides
•	(write down which barcode is added to which sample!!)
o	4 µl of dNTP, 1 µl of primer (2.5 mM) was added to the tailed RNA product
o	Incubation for 5 min at 70°C on heatblock (annealing)
o	Incubation on ice for 5 min
o	10 µl of the master mix of the reaction mix was added to each sample
•	Total volume of each sample is 20 µl at this point
•	Reaction mixture for 1 sample: 3 µl H2O, 4 µl 5xbuffer, 1 µl DTT, 1 µl RNaseOUT (40U), 1 µl Superscript  III
o	Incubate on heatblock for 50 min at 50°C (program 2)
•	DNA was purified using ethanol precipitation
o	See above
o	(400 µl liquid + 890 µl ice cold ethanol + 40 µl 3M NaAc pH 5.2 + 4 µl glycogen at -20°C for 30 min, 14000 rpm 10 min, 500 µl 70% ethanol wash, 14000 rpm 5 min)
•	Purified DNA fragments were resuspended in 10 µL H2O in 1.5 ml Eppendorf tubes
o	Add 10 µl loading dye to samples
•	Size selection
o	Run RT products on a 10% polyacrylamide TBE-urea gel as described above
•	Expected products of 93 nt (riboseq) and 113 nt (RNAseq)
o	cDNA was recovered O/N using 400 uL DNA gel elution buffer (
•	Prepared as 10 ml stock solution
•	300 mM NaCl (0.6 ml of 5M NaCl; use from Saisai), 1 mM EDTA (20 µl of 0.5 M EDTA; use from Saisai )
•	Diluted to 10 ml with RNase-free H2O
•	different buffer than used for first size selection
   
RNAseq samples (left) & Riboseq samples (right)
 
Day 6
•	gel debris was removed using a Spin-X column (Corning) 
o	12000 rpm 2 min
•	Eluent was transferred to a new 1.5 mL tube 
•	DNA was purified using ethanol precipitation
o	See above
o	(400 µl liquid + 890 µL ice cold ethanol + 40 µl 3M NaAc pH 5.2 + 4 µL glycogen at -20°C for 30 min, 14000 rpm 10 min, 500 µL 70% ethanol wash, 14000 rpm 5 min)
•	Purified DNA fragments were resuspended in 15 µL H2O in and transferred to PCR tubes
•	Circularization
o	First-strand cDNA was circularized in 20 µL of reaction containing DNA (15 µl), CircLigase buffer (2 µl), 50 mM MnCl2 (1 µl), 1mM ATP (1 µl) and CircLigase II (1 µl) (Epicentre)
o	Incubation in PCR machine of 1.5h at 60°C
•	Samples were transferred to 1.5 ml Eppendorf tubes
•	DNA was purified using ethanol precipitation
o	See above
o	(400 µl liquid + 890 µL ice cold ethanol + 40 µl 3M NaAc pH 5.2 + 4 µL glycogen at -20°C for 30 min, 14000 rpm 10 min, 500 µl 70% ethanol wash, 14000 rpm 5 min)
•	Purified DNA fragments were resuspended in 10 µL H2O 
•	PCR amplification
o	Mixture for 1 sample: H2O (5 µl), 5 x HF buffer (5 µl), 1 mM dNTP (0.5 µl), 10 mM qNTI200 primer (1.25 µl), 10 mM qNTI201 primer (1.25 µl), template DNA (5 µl) Phusion polyE (0.25 µl)
•	Store remaining template DNA at -20°C
•	qNTI200 = (5’-CAAGCAGAAGACGGCATA-3’)
•	qNTI201 =(5’-AATGATACGGCGACCACCGACAGGTTCAGAGTTCTACAGTCCGACG-3’)
o	Procedure: 98°C 30sec; 98°C 10sec, 60°C 20sec, 72°C 10sec for 12 cycles; 72°C 10min.
•	Program name “old seq”
•	Turn of hot start!
•	Estimate number of cycles (10-14x) based on amount of DNA
•	Size Selection
o	PCR products (should not be heated at 70°C) were separated on a non-denaturing 8% polyacrylamide TBE gel at 160V for 1 hour. Expected DNA at 120 bp was excised and recovered as described above. 
   
RNAseq samples (left) & Riboseq samples (right)
 
Day 7
•	gel debris was removed using a Spin-X column (Corning) 
o	12000 rpm 2 min
•	Eluent was transferred to a new 1.5 mL tube 
•	DNA was purified using ethanol precipitation
o	See above
o	(400 µl liquid + 890 µL ice cold ethanol + 40 µl 3M NaAc pH 5.2 + 4 µL glycogen at -20°C for 30 min, 14000 rpm 10 min, 500 µL 70% ethanol wash, 14000 rpm 5 min)
•	Purified DNA fragments were resuspended in 18 µL H2O in and transferred to PCR tubes
•	Deep Sequencing
o	Measure DNA concentration with Nanodrop
o	Label Eppendorf tubes
 
o	Log in to Cornell Biotechnology Resource Center (BRC)
•	https://cores.lifesciences.cornell.edu/userdev/index.php
•	Menu → Place order → Facility: illumina sequencing; Service: illumina sequencing → Create new order → Select payment method or enter new billing information → submit → Instrument: HiSEQ2500 → Run length: 50 bp → Sample preparation: Library submission (customer prep) → Sequencing Primer Type: Illumina TruSeq → Barcode type: Single Barcode → Multiplexed: No → Digital PCR on each sample before pooling: No or Yes → Organism: Eukaryotic → Submit
o	Bring samples to sequencing lab
•	http://www.biotech.cornell.edu/sites/default/files/uploads/Genomics/Illumina%20Submission%20Guidelines2.pdf
o	General info
•	http://www.biotech.cornell.edu/brc/genomics-facility/services/next-generation-sequencing

